### ISSUE TITLE
iPhone Mirroring [enhancement]

### ISSUE THREAD

**Issue by SoeYanPaing** [noah-nuebling/Mac-Mouse-Fix#1189]
## Description
Could you please look into improving the third-party mouse functionality on the iPhone Mirroring feature? Currently, it doesn’t scroll smoothly and is almost completely unusable.

## Rationale
This would solve the issue whenever I need to use iPhone remotely with any other mouse than Apple Magic Mouse (which sucks). 

## Additional info
–

**Comment by Cookdao** [comment]
Confirm

**Comment by Caffeine19** [comment]
No, it didn't work now. Don't know why it would work several minutes ago.

---

Did you turn off the trackpad simulation? It would make scrolling much smoother.

<img width="497" height="293" alt="Image" src="https://github.com/user-attachments/assets/17135690-c778-44a0-8f0a-dc3ac1a8bf91" />


---

Update again, It works now.
